contracts/VaultV3.vy

```diff
@@ -298,12 +298,11 @@ def initialize(
     """
     assert self.asset == empty(address), "initialized"
     assert asset != empty(address), "ZERO ADDRESS"
+    assert role_manager != empty(address), "ZERO ADDRESS"
 
     self.asset = asset
     # Get the decimals for the vault to use.
-    decimals: uint256 = convert(ERC20Detailed(asset).decimals(), uint256)
-    assert decimals < 256 # dev: see VVE-2020-0001
```

Comment:
curious what changed here to remove this?

Reply:
`this part of code doesn't need to convert uint8 to uint256 and check that it's not greater than border, it's useless. In times of VV#-2020-0001 uint8 type didn't exist, but now it exists in vyper.`